joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/Py2CpgOnFileSystem.scala

```diff
@@ -93,6 +97,22 @@ class Py2CpgOnFileSystem extends X2CpgFrontend[Py2CpgOnFileSystemConfig] {
     }
   }
 
+  private def filterEnvDir(config: Py2CpgOnFileSystemConfig, file: Path, inputPath: Path): Boolean = {
```

Comment:
just nit-picking, but I find this function a bit hard to follow: I had to look at the place where this is called to figure out what the return values actually mean. I think it might be clearer if we flip the returned bool around and rename the function to `isAutoDectedVenv`? (And adjust the call-site to use `filterNot`.)

Reply:
I wanted to keep it consistent to `filterIgnoreDirNames` down below (also their callsites).
But you are right, I guess both could benefit from a small refactoring there for readibility.

Reply:
I implemented the refactoring you suggested.